facility-locator/map.js

```diff
@@ -13,7 +13,9 @@ FacilityLocator = (function() {
       directionsStart,
       geolocated = false; // Used when determining starting location for driving directions
       zoomLevelToShowMarkers = 5;
-      infowindows = {};
+      infowindows = {},
```

Comment:
Pushed another commit that changes them to commas. They should be commas. Thanks for catching this.